File: app/src/main/java/com/skyline/app/PrivacyActivity.java
package com.skyline.app;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import com.skyline.app.databinding.ActivityPrivacyBinding;

public class PrivacyActivity extends AppCompatActivity {

    private ActivityPrivacyBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityPrivacyBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        setupClicks();
    }

    private void setupClicks() {
        binding.btnBack.setOnClickListener(v -> finish());

        binding.btnContactSupport.setOnClickListener(v -> {
            startActivity(new Intent(PrivacyActivity.this, SupportActivity.class));
        });
    }
}

File: app/src/main/java/com/skyline/app/PrivacyActivity.kt
package com.skyline.app

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.skyline.app.databinding.ActivityPrivacyBinding

class PrivacyActivity : AppCompatActivity() {

    private lateinit var binding: ActivityPrivacyBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityPrivacyBinding.inflate(layoutInflater)
        setContentView(binding.root)

        setupClicks()
    }

    private fun setupClicks() = with(binding) {
        btnBack.setOnClickListener {
            finish()
        }

        btnContactSupport.setOnClickListener {
            startActivity(Intent(this@PrivacyActivity, SupportActivity::class.java))
        }
    }
}
